lib/httpClient/projections/getInfo.js

```diff
@@ -1,14 +1,20 @@
-import debugModule from 'debug';
 import assert from 'assert';
+import axios from 'axios';
+import debugModule from 'debug';
 
 const debug = debugModule('geteventstore:getProjectionInfo');
 const baseErr = 'Get Projection Info - ';
 
-export default (getAllProjectionsInfo) => async (name) => {
+export default (config, getAllProjectionsInfo) => async (name) => {
 	assert(name, `${baseErr}Name not provided`);
 
 	const projectionsInfo = await getAllProjectionsInfo();
 	const projectionInfo = projectionsInfo.projections.find(projection => projection.name === name);
 	debug('', 'Projection Info: %j', projectionInfo);
-	return projectionInfo;
+
+	const configDetails = await axios.get(`${config.baseUrl}/projection/${name}/config`);
+	debug('', 'Details: %j', configDetails.data);
+	const projectionConfig = configDetails.data
+
+	return { ...projectionInfo, ...{ config: projectionConfig } };
```

Comment:
Can you please move this config call to its own method: `projections.config`?  And then extend this method (`getInfo`) to have a boolean `includeConfig` where we can extend the `info` results?